src/renderer/store/settings.store.ts

```diff
@@ -333,6 +335,8 @@ const initialState: SettingsState = {
     general: {
         accent: 'rgb(53, 116, 252)',
         albumArtRes: undefined,
+        albumBackground: false,
+        albumBackgroundBlur: 1.5,
```

Comment:
One more, since I forgot to add this as part of review. I think it would make sense to default to 6 to preserve the similar behavior with just a single color. I do appreciate that this is configurable though.